runtime/services/multimediad/service.js

```diff
@@ -154,18 +166,18 @@ MultiMedia.prototype.listenEvent = function (player, appId) {
         logger.error(`try to stop player error with appId: ${appId}`, error.stack)
       }
     }
-    this.emit('playbackcomplete', '' + player.id)
+    this.emit('playbackcomplete', '' + player.id, '' + player.duration, '' + player.position)
```

Comment:
Could number be passed through DBus? Does `dbus` module in ShadowNode support it?

Reply:
ShadowNode covers sending numbers at [test/run_pass/test_dbus_service_and_call.js#L22](https://github.com/Rokid/ShadowNode/blob/master/test/run_pass/test_dbus_service_and_call.js#L22), as for signal, it uses the same encoding function [`iotjs_dbus_encode_jobject `](https://github.com/Rokid/ShadowNode/blob/master/src/modules/iotjs_module_dbus.c#L172), so I think it might be Ok for this case.